### PR TITLE
Rocky 8 support

### LINKED ISSUE #144: FreeRADIUS stopped working after a recent upgrade on RHEL 8
Hi,

After a FreeRADIUS upgrade on RHEL 8 (freeradius-3.0.20-3.module+el8.3.0+7597+67902674.x86_64), FreeRADIUS stopped working with this error in the logs:

```
Systemd start for radiusd failed!
journalctl log for radiusd:
-- Logs begin at Tue 2020-12-29 04:36:26 CET, end at Tue 2021-01-12 18:22:30 CET. --
Jan 12 18:22:30 <redacted> systemd[1]: Starting FreeRADIUS high performance RADIUS server....
Jan 12 18:22:30 <redacted> sh[1960192]: /bin/sh: /etc/raddb/certs/bootstrap: No such file or directory
Jan 12 18:22:30 <redacted> systemd[1]: radiusd.service: Control process exited, code=exited status=127
Jan 12 18:22:30 <redacted> systemd[1]: radiusd.service: Failed with result 'exit-code'.
Jan 12 18:22:30 <redacted> systemd[1]: Failed to start FreeRADIUS high performance RADIUS server..
```

I guess this is due to https://github.com/djjudas21/puppet-freeradius/blob/e1dec8c74515d8c15cefc2385fbe2075fef63690/manifests/init.pp#L88

What would be the best solution, keeping this file, or dropping a systemd override to prevent the service from calling this script as part of its ExecStartPre command?

### PR DESCRIPTION
**Pull request by djjudas21**
This PR includes the changes by @minorOffense from #178 which was incomplete, and some fixes by @nward from the `144-rocky8` branch.

Supersedes #178 

Fixes #144

### CODE REVIEW

File: manifests/init.pp
Comment (nward) [inline manifests/init.pp:113]: Let's remove this todo / create an issue in its place?

Comment (nward) [reply manifests/init.pp:113]: Or maybe we don't need to as this is only on el8, so it's always going to be radius.service..

**Review by nward — DISMISSED** [review]
LGTM I reckon!

**Comment by nward** [comment]
Want me to have a poke at the PDK tests failing, @djjudas21 ? I am deep in puppet brain right at the minute, and just finished something else, so now is a good time..

**Comment by djjudas21** [comment]
Yes please @nward!

**Comment by djjudas21** [comment]
I updated this branch after merging #196 but there are still problems @nward

**Comment by nward** [comment]
OK - fixed, removed the duplicate dependency. I also rebased on main rather than a merge (keeps the history tidier), hence the force push :)

**Review by nward — APPROVED** [review]
LGTM - we should squash+merge as there's some reworking going on here which confuses the history a bit. Can we do that while keeping credit for the changes to @minorOffense? If GitHub doesn't let us I can do that locally and force push.

**Comment by djjudas21** [comment]
I suspect using the inbuilt squash + merge will attribute the resulting commit to the person who clicks the button

**Review by nward — APPROVED** [review]
OK @djjudas21 I've added some tests - good to merge if you are happy with this.

**Comment by djjudas21** [comment]
If you're happy and it's passing tests, then I'm happy! Will merge now

**Pull request merged**
